odinclient/src/main/kotlin/me/odinclient/features/impl/skyblock/AutoHarp.kt

```diff
@@ -0,0 +1,68 @@
+package me.odinclient.features.impl.skyblock
+
+import me.odinmain.features.Category
+import me.odinmain.features.Module
+import me.odinmain.utils.skyblock.LocationUtils.inSkyblock
+import net.minecraft.client.gui.inventory.GuiChest
+import net.minecraft.init.Blocks
+import net.minecraft.inventory.ContainerChest
+import net.minecraft.item.ItemBlock
+import net.minecraftforge.client.event.GuiOpenEvent
+import net.minecraftforge.client.event.GuiScreenEvent
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
+
+/**
+ * Module to automatically do the Melody's Harp minigame.
+ *
+ * Modified from: https://github.com/FloppaCoding/FloppaClient/blob/master/src/main/kotlin/floppaclient/module/impl/misc/AutoHarp.kt
+ *
+ * @author Aton
+ */
+object AutoHarp : Module(
+    "Auto Harp",
+    category = Category.SKYBLOCK,
+    description = "Automatically Completes Melody's Harp"
+){
+    private var inHarp = false
+    private var lastInv = 0
+
+    @SubscribeEvent
+    fun onGuiOpen(event: GuiOpenEvent) {
+        if (event.gui !is GuiChest || !inSkyblock) return
+        val container = (event.gui as GuiChest).inventorySlots
+        if (container is ContainerChest) {
+            val chestName = container.lowerChestInventory.displayName.unformattedText
```

Comment:
use ContainerChest.name extension value from Utils.kt, container.name in this case

Reply:
do the same as you do later on in the code, container.name since its known its a ContainerChest